lessons/Early-Math/absolute-value-and-opposites/langs/en-us/index.adoc

```diff
@@ -0,0 +1,150 @@
+= Absolute Value and Opposites
+
+@description{Students use Circles of Evaluation to analyze the effect of negation and absolute value.}
+
+@lesson-prereqs{distributive-property}
+
+@keywords{Opposite, Absolute Value}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Define absolute value as a number's distance from zero on the number line.
+* Recognize that two numbers are referred to as "opposites" if they are the same distance from zero.
+* Use Circles of Evaluation to analyze expressions with opposite and absolute value.
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to think about opposites and absolute value.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
```

Comment:
<img width="441" alt="Screen Shot 2023-01-16 at 1 48 21 PM" src="https://user-images.githubusercontent.com/66691020/212755786-5d3a400c-3dd4-42cb-a362-eaa310e4e290.png">
Do you prefer ampersand or _and_? Be consistent.